Assignment2/FORGIS98_fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,66 @@
+{
+  "@context" : {
+    "base" : "http://example.org/",
+    "data" : "http://example.org/data",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "includes" : "rdfs:label",
+    "hasMeasurement" : "rdfs:label",
+    "hasOwner" : "rdfs:label",
+    "hasTemperature" : "rdfs:label",
+    "atTime" : "rdfs:label",
+    "hasName" : "rdfs:label",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@graph" : [
+    {
+      "@id" : "base:Class01",
+      "@type" : "base:",
```

Comment:
You have to specify the type of each object. In this case, it might be "base:Class01". The way you defined it generates only "http://example.org/" but machines need to distinguish between objects.
Define types for the others (Sensor, Computer, etc.)